lsm/lsm.h

```diff
@@ -37,13 +37,40 @@ class maybe_value {
   explicit maybe_value(int32_t _v, char _deleted) : v{_v}, is_deleted{_deleted} {}
 };
 
+class Level {
+public:
+  std::vector <File> files;
+  const int max_size = 2;
+  int index;
+  std::filesystem::path level_directory;
+
+  explicit Level(int index, std::filesystem::path _directory) :
+      level_directory{(std::filesystem::path) _directory / std::to_string(index)} {
+    if (std::filesystem::exists(level_directory)) {
+      std::cerr << "WARNING: " << level_directory << " already exists.";
+    }
+    this->index = index;
```

Comment:
instead of using `this->foo = foo` in the constructor body, use the initializer list and use a distinct name:
```cpp
  explicit Level(int _index, std::filesystem::path _directory) :
       level_directory{(std::filesystem::path) _directory / std::to_string(index)},
       index{_index} {

```